app/app.css

```
@@ -1,15 +1,45 @@
+@import url('//cdn.jsdelivr.net/gh/neodgm/neodgm-webfont@1.600/neodgm/style.css');
+@import url("https://font.elice.io/css?family=Elice+Digital+Baeum");
 @import "tailwindcss";
 
+/* 
 @theme {
   --font-sans: "Inter", ui-sans-serif, system-ui, sans-serif,
     "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
-}
+} */
```

[@theme](https://tailwindcss.com/docs/font-family)을 그대로 활용할 수 있을 것 같은데, 확인 한번 부탁드려요~!

```suggestion
@import "tailwindcss";

@theme {
  --font-apple-gothic: 'Apple SD Gothic Neo', sans-serif; 
  --font-sf: 'SF Pro Display', sans-serif;
  ...
}
```

---

살려두었습니다 ~